constants/fluid.py

```diff
@@ -0,0 +1,2 @@
+vaultResolver = "0x56ddF84B2c94BF3361862FcEdB704C382dc4cd32"
```

Comment:
https://etherscan.io/address/0x49290f778faAD125f2FBCDE6F09600e73bf4bBd9 is the newest one